src/aks-preview/azext_aks_preview/vendored_sdks/azure_mgmt_preview_aks/v2024_07_02_preview/aio/operations/_agent_pools_operations.py

```diff
@@ -1032,7 +1032,7 @@ async def get_available_agent_pool_versions(
         """Gets a list of supported Kubernetes versions for the specified agent pool.
 
         See `supported Kubernetes versions
-        <https://docs.microsoft.com/azure/aks/supported-kubernetes-versions>`_ for more details about
+        <https://learn.microsoft.com/azure/aks/supported-kubernetes-versions>`_ for more details about
```

Comment:
The content under path `vendored_sdks` is generated based on swagger. I think you'll need to change the content of source, otherwise these changes will be overwritten when the new SDK is generated.

Reply:
@xmdanni the files under vendored_sdk_xxx folder are auto generated from https://github.com/Azure/azure-rest-api-specs/tree/main/specification, which also needs an update for microsoft.com domain